scripts/V278__DOCUMENTO.sql

```diff
@@ -53,10 +54,18 @@ select
 		'FOREIGN KEY (usuario_id) REFERENCES usuario (id)'
 	);
 
+select
+	f_cria_fk_se_nao_existir(
+		'documento',
+		'documento_ue',
```

Comment:
coloque o _fk para ficar no padrão
```suggestion
		'documento_ue_fk',
```